frontend_vue/src/router/index.ts

```diff
@@ -41,6 +40,14 @@ const router = createRouter({
         title: 'Manage Canarytoken',
       },
     },
+    {
+      path: '/manage/:tokentype',
```

Comment:
this seems too similar to the standard manage path. we have `path: '/manage/:auth/:token',` and now `path: '/manage/:tokentype',` but to me those two clash. the second param is :auth in one and :tokentype in the other. can we give this another pathname unique to the custom flow?